truffle/src/main/java/org/jruby/truffle/pack/nodes/read/ReadDoubleNode.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Indeed, my mistake, I meant integer write nodes. But it's also good to keep one per file for consistency then.